packages/js/assets/user-feedback-form.js

```diff
@@ -0,0 +1,122 @@
+(function (window, document) {
+  // eslint-disable-next-line quotes
+  const TEMPLATE = `$$TEMPLATE$$`
```

Comment:
The HTML template is in a separate file for better readability and maintainability. A post-build script merges the js and html files into one to be served from a script tag (see `generate-feedback-form-assets.js`).